src/main/scala/org/apache/spark/ml/param/PythonWrappableParam.scala

```diff
@@ -4,20 +4,27 @@
 package org.apache.spark.ml.param
 
 import spray.json._
+import java.lang.{StringBuilder => JStringBuilder}
+
+trait PythonPrinter extends CompactPrinter {
+  override protected def printLeaf(x: JsValue, sb: JStringBuilder): Unit = {
+    x match {
+      case JsNull      => sb.append("None")
+      case JsTrue      => sb.append("True")
+      case JsFalse     => sb.append("False")
+      case JsNumber(x) => sb.append(x)
```

Comment:
any potential local issues?

Reply:
what do you mean?

Reply:
writeing a float number like 1.2 will be converter to string locale dependent. e.g. DE_de will get you 1,2

decimal/thousand separator are exactly opposite to EN_us